core/shaders/point.vs

```diff
@@ -33,6 +35,9 @@ attribute LOWP vec4 a_stroke;
 attribute float a_scale;
 #else
 attribute vec3 a_position;
+attribute vec4 a_outline_color;
+attribute float a_outline_edge;
+attribute float a_aa_factor;
```

Comment:
If I understand this parameter, it seems like `a_aa_factor` could be a uniform instead of an attribute.